### ISSUE TITLE
Navigation bar: User dropdown menu not accessible via tabbing

### ISSUE THREAD

**Issue by alisan16** [department-of-veterans-affairs/caseflow#12242]
In all applications the user name in the upper right hand corner is not keyboard accessible (New: 9/5/2019)  below is the image of it being clicked on via mouse.  

<img width="1094" alt="Screen Shot 2019-09-27 at 5 12 33 PM" src="https://user-images.githubusercontent.com/1198106/65802555-05a46480-e14a-11e9-9a4f-25c9b6822935.png">

**Comment by lomaxap** [comment]
@alisan16 is this a duplicate of #12122? if so, @aplybeah thinks that [this PR](https://github.com/department-of-veterans-affairs/caseflow-frontend-toolkit/pull/53) in the frontend toolkit resolves this

**Comment by alisan16** [comment]
@lomaxap This one is about the user dropdown menu on the top right, #12122 is about the product dropdown on the top left

<img width="1036" alt="Screen Shot 2019-10-30 at 3 00 33 PM" src="https://user-images.githubusercontent.com/1198106/67889890-0b64df80-fb26-11e9-87ad-d42856e362b8.png">

**Comment by aplybeah** [comment]
@alisan16 Both of the dropdown menus in the navigation bar are rendered by the `DropdownMenu` component in the front-end toolkit. I believe that the current hash being used to connect the front-end to Caseflow isn't the branch where these changes actually are.
<img width="1675" alt="Screen Shot 2019-10-31 at 11 12 19 AM" src="https://user-images.githubusercontent.com/37313082/67959634-71597180-fbcf-11e9-9fb0-9a726ccd00c4.png">

Steps to replicate:

- in `client/package.json` change the hash at the end of the front-end-toolkit from `3ef2bfb` to `a0279be`
- run `yarn install` to update the package

I will take a closer look at this however, because the screen reader is reporting the items as 'visited links'

**Comment by alisan16** [comment]
@aplybeah Thanks for clarifying! That is great that it's the same component. Can we update the front-end-toolkit hash being used to load Caseflow?

**Comment by aplybeah** [comment]
@alisan16 It's possible, but I'd like to check with someone like @jcq first.

**Comment by jcq** [comment]
@aplybeah You should be good to go to do a PR for main caseflow repo that updates the hash for the caseflow-frontend-toolkit. Bump that hash, update w/ `yarn`, verify that everything is working, and should be all set!

**Comment by leikkisa** [comment]
Between the conversations above, and that menu being fully keyboard accessible to me now, I believe this issue can be closed.
